website/docs/data-display/bar-distribution/examples/clickable.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Plot, DistributionBar } from 'intergalactic/d3-chart';
+import { scaleLinear, scaleBand } from 'd3-scale';
+import { Flex } from 'intergalactic/flex-box';
+import { Text } from 'intergalactic/typography';
+
+const Demo = () => {
+  const MARGIN = 30;
+  const width = 500;
+  const height = 500;
+
+  const sum = data.reduce((acc, d) => acc + d.value, 0);
+
+  const xScale = scaleLinear()
+    .range([0, width])
+    .domain([0, sum]);
+
+  const yScale = scaleBand()
+    .range([height - MARGIN, MARGIN])
+    .domain([...data].reverse().map((d) => d.category))
+    .paddingInner(0.6)
+    .paddingOuter(0.2);
+
+  return (
+    <Plot data={data} scale={[xScale, yScale]} width={width} height={height}>
+
+      <DistributionBar x='value' y='category'>
+        <DistributionBar.Hover />
+        <DistributionBar.Tooltip>
+          {({ index }) => {
+            return {
+              children: (
+                <>
+                  <DistributionBar.Tooltip.Title>{data[index].category}</DistributionBar.Tooltip.Title>
+                  <Flex justifyContent='space-between'>
+                    <DistributionBar.Tooltip.Dot mr={4}>Bar</DistributionBar.Tooltip.Dot>
+                    <Text bold>{data[index].value}</Text>
+                  </Flex>
+                </>
+              ),
+            };
+          }}
+        </DistributionBar.Tooltip>
+        <DistributionBar.Annotation>
+          <DistributionBar.Label />
+          <DistributionBar.Percent />
+          <DistributionBar.Value />
+        </DistributionBar.Annotation>
+        <DistributionBar.Bar onClick={(barIndex) => alert(`Bar ${barIndex} clicked`)}>
```

Comment:
Maybe we could show clickable example with simple usage instead of advanced?

Also, I'm not sure, that we should show advanced examples... This is really very simple component, which could have typed data (maybe with handlers or links). And with advanced examples we'll loose control how people are using it.